src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -115,7 +115,6 @@
     <!--  Used for disclaimers  -->
     <Style x:Key="DisclaimerStyle"
            TargetType="TextBlock">
-        <Setter Property="FontStyle" Value="Italic" />

```

Comment:
We might want to replace this with
```suggestion
        <Setter Property="Foreground" Value="#CCCCCC" />
```

Reply:
We should avoid using any hardcoded colors, and instead prefer the XAML color ramp.

Reply:
Oh right, I should've looked at the screenshots more closely. You're right that it doesn't look quite as nice as your current approach.